sw/device/silicon_creator/lib/drivers/alert_unittest.cc

```diff
@@ -39,10 +46,18 @@ TEST_F(InitTest, AlertConfigureAlertBadEnable) {
   // experience an error when evaluating the enable parameter.
   EXPECT_SEC_WRITE32_SHADOWED(
       base_ + ALERT_HANDLER_ALERT_CLASS_SHADOWED_0_REG_OFFSET, 0);
-  EXPECT_EQ(alert_configure(0, kAlertClassA, (alert_enable_t)-1),
+  EXPECT_EQ(alert_configure(0, kAlertClassA, static_cast<alert_enable_t>(-1)),
             kErrorAlertBadEnable);
 }
 
+TEST_F(InitTest, LocalAlertConfigureAlertBadEnable) {
+  EXPECT_SEC_WRITE32_SHADOWED(
+      base_ + ALERT_HANDLER_LOC_ALERT_CLASS_SHADOWED_0_REG_OFFSET, 0);
+  EXPECT_EQ(
+      alert_local_configure(0, kAlertClassA, static_cast<alert_enable_t>(0)),
```

Comment:
(just to be consistent with the test above)

```suggestion
      alert_local_configure(0, kAlertClassA, static_cast<alert_enable_t>(-1)),
```

Reply:
done